packages/core/src/server/index.tsx

```diff
@@ -27,7 +27,13 @@ export default ({ packages }): ReturnType<Koa["callback"]> => {
   const app = new Koa();
 
   // Serve static files.
-  app.use(mount("/static", serve("./build/static")));
+  app.use(async (ctx, next) => {
+    const moduleStats = await getStats({ target: "module" });
+    const es5Stats = await getStats({ target: "es5" });
+    const stats = moduleStats || es5Stats;
+    const publicPath = stats ? stats.publicPath : "/static";
+    return mount(publicPath, serve("build/static"))(ctx, next);
```

Comment:
is the `build/static` the going to refer to the same path as the `./build/static` from before the change?

Reply:
With `--publicPath` you change the path where those files are served (the URL) but they are always stored in the same folder (`./build/static`).

I don't know if this answer your question. 😅

Reply:
Yes, I used an internal variable named `outDir` for the `/build` folder, but right now there's no way to change it from the outside. It's there waiting until we release the `frontity.config.js` file or we add a `--buildDirectory` option.

Reply:
oh, I meant are there any circumstances where `build/static` can point to a different path than `./build/static`, but I've realized that they're exactly the same: https://unix.stackexchange.com/a/397526